README.md

```diff
@@ -198,6 +198,9 @@ You can also load your JSONified config in from an ENV variable (e.g. `KUBE_CONF
 ```
 Kubeclient::Config.new(JSON.parse(ENV['KUBE_CONFIG']), nil)
 ```
+###Supported kubernetes versions
+
+For 1.1 only the core api v1 is supported, apis/extensions are supported in later versions.
```

Comment:
apis/extensions is only one group api(the only one in 1.1 but not in later versions) please change to 
`For 1.1 only the core api v1 is supported, all api groups are supported in later versions.`